zjit/src/asm/mod.rs

```diff
@@ -208,6 +208,14 @@ impl CodeBlock {
         self.dropped_bytes
     }
 
+    /// Set false to dropped_bytes if the current zjit_alloc_bytes() + code_region_size
+    /// is below --zjit-mem-size.
+    pub fn check_remaining_bytes(&mut self) {
```

Comment:
`check` seems like a read operation. This can write. Maybe `update_dropped_bytes` or similar?

Reply:
renamed it to `update_dropped_bytes` https://github.com/ruby/ruby/pull/15069/commits/c0b5b07fe71c3cc4f63aa8c2a290cf6911504bc6